pkg/internal/controllers/certificaterequestpolicies.go

```diff
@@ -118,8 +118,8 @@ func addCertificateRequestPolicyController(_ context.Context, opts Options) erro
 
 	return ctrl.NewControllerManagedBy(opts.Manager).
 		For(new(policyapi.CertificateRequestPolicy)).
-		Watches(&source.Channel{Source: genericChan}, handler.EnqueueRequestsFromMapFunc(
-			func(obj client.Object) []reconcile.Request {
+		WatchesRawSource(&source.Channel{Source: genericChan}, handler.EnqueueRequestsFromMapFunc(
```

Comment:
Since `Watches` now accepts `client.Object` only we here need to use the new lower level `WatchesRawSource` that has the same signature as `Watches` had up till now https://github.com/kubernetes-sigs/controller-runtime/pull/2135/commits/dc62ed5dadc6ea2d3e22546368b939a2db1edfb2#diff-e77b9468ab935d5ea4d5cdae3b994114bada17df0001570e4b1436419afd50ccR176-R179